### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by jeffreyladd**
# Summary
Updates the FAQ Page for one added question and also some other tweaks

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Instead, we recommend using either the latest LTS release or most recent stable
 To troubleshoot, please ensure that:
 
 * You are using the latest version of DSharpPlus.
-* You have properly enabled VoiceNext with your instance of `DiscordClient`.
+* You have properly enabled VoiceNext with your instance of @DSharpPlus.DiscordClient .
```
Comment (Neuheit) [inline docs/faq.md:28]:
```suggestion
* You have properly enabled VoiceNext with your instance of @DSharpPlus.DiscordClient.
```

File: docs/faq.md [hunk 2]
```diff
@@ -72,11 +72,15 @@ However, if you meant an activity like this:
 
 You can use either of the following
 
-* The overload for `DiscordClient#ConnectAsync()` which accepts a `Discord​Activity`.
-* `DiscordClient#UpdateStatusAsync()` at any point after `Ready` has been fired.
+* The overload for @DSharpPlus.DiscordClient.ConnectAsync(DiscordActivity,System.Nullable{UserStatus},System.Nullable{DateTimeOffset}) which accepts a @DSharpPlus.Entities.DiscordActivity.
+* @DSharpPlus.DiscordClient.ConnectAsync(DiscordActivity,System.Nullable{UserStatus},System.Nullable{DateTimeOffset}) at any point after `Ready` has been fired.
```
Comment (Neuheit) [inline docs/faq.md:76]: Is this meant to be `UpdateStatusAsync()` instead? If so, also add a link to the sharded client method.

Comment (jeffreyladd) [reply docs/faq.md:76]: Ill fix this tonighy, i copied and pasted the wrong xref

**Comment by jeffreyladd** [comment]
This is ready for review/merge and my changes can be viewed on https://jeffreyladd.github.io/faq.html

**Review by Neuheit — APPROVED** [review]

**Pull request merged**